.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: check-yaml
+        exclude: tests/data_context/fixtures/bad_yml/great_expectations.yml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+-   repo: https://github.com/psf/black
+    rev: 19.10b0
+    hooks:
+    -   id: black
+- repo: https://github.com/timothycrosley/isort
+  rev: 4.3.21-2
+  hooks:
+      - id: isort
```

Comment:
Might want to install a plugin for your editor to automatically include new lines at the end of file if they don't exist. There's actually a (built-in) pre-commit hook that does this for you 😉  see below for an example

https://github.com/quantumblacklabs/kedro/blob/9f95ee680012f7a20c31fcc820b7ed3a894ae9e6/.pre-commit-config.yaml#L11